ci-operator/config/openshift/release/openshift-release-master__nightly-4.16.yaml

```diff
@@ -827,6 +827,20 @@ tests:
       - observers-resource-watch
     workflow: telco5g-hcp-cnf
 - as: metal-ovn-network-flow-matrix-bm
+  cluster: build05
+  cron: 0 23 * * 1
+  steps:
+    cluster_profile: equinix-ocp-metal
+    env:
+      CLUSTERTYPE: baremetal
+    observers:
+      enable:
+      - observers-resource-watch
+    test:
+    - ref: network-flow-matrix-tests
```

Comment:
How long does this step take? I wonder if we can just add it to an existing job rather then add a new job?

Reply:
This step is max 20 min, BUT it configures a firewall so might break other tests, that's why we want is to be separated for a a period of time to ensure stability

Reply:
ok, makes sense